### DISCUSSION TITLE
请问现在抢票时会弹验证码吗？如果弹的话，会自动过验证吗？

### DISCUSSION THREAD

**Discussion by Violiate** [mikumifa/biliTickerBuy#94] [提问专区]
还是如果请求不频繁的话不会弹出来验证码的？

**Comment by mikumifa** [comment]
热门的票都有验证码

**Reply by Violiate** [reply]
那目前的软件会自动完成验证码吗？还是弹出来手动完成呢

**Reply by mikumifa** [reply]
短时间内不会，我测试过

**Reply by Violiate** [reply]
> 热门的票都有验证码

就是说在抢票前去训练验证码页面手动完成一次验证码，就不会弹出验证码的意思吗？还是说需要使用手机端找一个验证码进行完成呢？

**Reply by mikumifa** [reply]
不，抢的时候会弹出来，2者没有关系

**Reply by mikumifa** [reply]
具体流程看主页的那个gif

**Reply by Violiate** [reply]
明白了，开抢时候弹出来的话需要使用者自己拖动滑块进行拼接完成验证吗？

**Reply by mikumifa** [reply]
是的

**Reply by mikumifa** [reply]
训练那个只是帮你熟悉流程

**Comment by Violiate** [comment]
那有考虑接入自动完成验证码的api吗？因为做验证感觉好费时间，浪费的那几秒就失去了软件的优势，就和后续的人一起拥挤了

**Reply by mikumifa** [reply]
验证码api也要十几秒，没你想的那么离谱，后续会加，但bw之前是肯定不会完成的，我要测试没问题才会发布

**Reply by Violiate** [reply]
> 验证码api也要十几秒，没你想的那么离谱，后续会加，但bw之前是肯定不会完成的，我要测试没问题才会发布

可能是你用的那个api太慢了，我用的打b站的基本是一秒完成

**Reply by mikumifa** [reply]
给个链接，我试试

**Reply by Violiate** [reply]
> 给个链接，我试试

rrocr.com能过所有极验的

**Comment by mikumifa** [comment]
好好，阿里嘎都you

**Reply by mikumifa** [reply]
那太棒了，1055069518@qq.com 私

**Comment by Bocas1** [comment]
rrocr.com访问不了

**Reply by mikumifa** [reply]
不开梯子
